test/JDBC/expected/table_variable_xact_errors_isolation_snapshot.out

```diff
@@ -583,9 +583,9 @@ AS
     DELETE trgt FROM Discount trgt           -- Discount does not exist
     COMMIT
 go
-~~ERROR (Code: 2714)~~
+~~ERROR (Code: 33557097)~~
```

Comment:
Please map this error to 2714 as well. Even though error code is same, message is different so we need to add a new entry into mapping txt file.

Reply:
Mapped the error.